tools/pkg-config/stumpless.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
```

Comment:
I believe this should be `${prefix}/bin` based on the stumpless install target at line 953 of the CMakeLists.txt file?

Reply:
`pkg-config` is for libraries, never for binaries. The `exec_prefix` reflects the working directory/target. The naming is a bit confusing yeah.